plugins/kubernetes/api-report.md

```diff
@@ -421,6 +424,13 @@ export class ServerSideKubernetesAuthProvider
 // @public (undocumented)
 export const ServicesAccordions: ({}: ServicesAccordionsProps) => JSX.Element;
 
+// @public
+export const useCustomResources: (
+  entity: Entity,
+  customResourceMatchers: CustomResourceMatcher[],
+  intervalMs?: number,
```

Comment:
I'm feeling it'd be best to make this an options object tbh, as in

```ts
useCustomResource(entity, matcher, { intervalMs: 100 })
```

Otherwise it'll be a bit of a pita to add any additional options in the future.

Noticed this matches the existing `useKubernetesObjects` though, so feeling it's fine to ship either way and can be fixed in the future if needed. You can always go for `intervalMsOrOptions?: number | { ... }` in the future as well, to avoid a breaking change.